apps/src/assetManagement/AssetManager.jsx

```diff
@@ -135,7 +135,7 @@ module.exports = React.createClass({
 
       assetList = (
         <div>
-          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0', paddingRight: '15px'}}>
+          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0'}}>
```

Comment:
Note: In React, we can actually use a number for (most) styles, and let it append px for us, i.e. `{maxHeight: 330}`